VideoApi/VideoApi.DAL/Queries/GetConsultationRoomByIdQuery.cs

```diff
@@ -29,14 +29,17 @@ public GetRoomByIdQueryHandler(VideoApiDbContext context)
             _context = context;
         }
 
-        public Task<Room> Handle(GetRoomByIdQuery query)
+        public async Task<ConsultationRoom> Handle(GetConsultationRoomByIdQuery query)
         {
-            return _context.Rooms
+            var room = await _context.Rooms
                 .Include(x => x.RoomParticipants)
```

Comment:
could we use `.OfType<ConsultationRoom>` or not reeturn ConsultationRoom but just Room as it was?
If returing ConsultationRoom maybe the query name needs to be updated too?

Reply:
Yes, thanks! I knew there would be some gaps.

Reply:
done, i think the page didn't refresh at the time

Reply:
done